app/views/simulations/index.html.erb

```diff
@@ -1,22 +1,29 @@
-<table>
-    <thead>
-        <tr>
-            <th>Simulation</th>
-            <th>Name</th>
-            <th>Runs</th>
-            <th>Started</th>
-            <th>Finished</th>
-        </tr>
-    </thead>
-    <tbody>
-        <% @simulations.each do |simulation| %>
-            <tr>
-                <td><%= simulation.id %></td>
-                <td><%= link_to(simulation.name, simulation) %></td>
-                <td><%= simulation.runs %></td>
-                <td><%= simulation.start %></td>
-                <td><%= simulation.finish %></td>
-            </tr>
-        <% end %>
-    </tbody>
-</table>
+<div >
+    <div class="py-4">
+        <%= link_to "New simulation", new_simulation_path, class: "bg-blue-800 text-white px-5 py-2 rounded-full hover:bg-blue-400" %>
+    </div>
+    <div>
+        <table>
```

Comment:
```suggestion
<div class="w-full">
    <div class="py-4">
        <%= link_to "New simulation", new_simulation_path, class: "bg-blue-800 text-white px-5 py-2 rounded-full hover:bg-blue-400" %>
    </div>
    <div>
        <table class="table-auto">
```

These classes make the page look a bit better

**Before**
<img width="1172" alt="image" src="https://github.com/eargollo/soccer/assets/1158152/8557064e-6de4-4dd0-8ab9-e079862f326b">


**After**
<img width="1174" alt="Screenshot 2023-11-03 at 20 44 03" src="https://github.com/eargollo/soccer/assets/1158152/bfe4e493-b7e2-4d4e-b1ba-1e633172d5ea">

Reply:
Nice!!! Applied in #21 

Hummm... I should seed some data.

Reply:
My UX needs a tooooonnn of improvements 😉